plaidml/bridge/openvino/tests/functional/plugin/plaidml/shared_tests_instances/single_layer_tests/transpose.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "common_test_utils/test_constants.hpp"
+#include "single_layer_tests/transpose.hpp"
+
+using LayerTestsDefinitions::TransposeLayerTest;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+    InferenceEngine::Precision::FP32,
+};
+
+INSTANTIATE_TEST_CASE_P(UnsqueezeCheck, TransposeLayerTest,
```

Comment:
unsqueezed might be a typo here ? needs to be INSTANTIATE_TEST_CASE_P(TransposeCheck, TransposeLayerTest...

Reply:
That's a mistake, thanks!

Reply:
(It doesn't matter since this PR was marked as Draft, but don't approve PRs if you have corrections or concerns, even minor ones. Mergify will automatically merge approved commits, often before the author can make changes. And after the changes are in you'll have to approve again anyway.)